src/axom/slic/docs/sphinx/sections/architecture.rst

```diff
@@ -423,12 +429,13 @@ The ``MyStream`` class implements the ``LogStream::append()`` method of the
                           int line,
                           bool AXOM_UNUSED_PARAM(filtered_duplicates) )
    {
-      assert( m_stream != nillptr );
+      assert( m_stream != nullptr );
```

Comment:
Yikes! How did that compile?

Reply:
It didn't because it is just a code chunk in the documentation. A very good example why we shouldn't do this in general =)